config/manager/manager.yaml

```diff
@@ -37,10 +37,19 @@ spec:
           - configMapRef:
               name: ironic
         name: manager
+        securityContext:
+          allowPrivilegeEscalation: false
         livenessProbe:
           httpGet:
             path: /healthz
             port: 9440
-          initialDelaySeconds: 3
-          periodSeconds: 3
+          initialDelaySeconds: 15
```

Comment:
@kashifest did you also change these value or were part of the automatic update?

Reply:
These were also auto generated.